config/peachpayment.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-if (config('app.env') === 'production') {
+if (in_array(config('app.env'), ['production', 'uat'])) {
```

Comment:
I don't think we should include this - it's quite specific to our project

Reply:
Suspected as much